css/style.css

```diff
@@ -1 +1,1091 @@
+}
+
+.about * {
+  background-color: transparent;
+}
+
+.about h2 {
+  margin-bottom: 16px;
+}
+
+.about p {
+  margin: 0 auto;
+  font-family: var(--text-font);
+  color: var(--text-color);
+  width: 930px;
+}
+
+.about-list {
+  margin-top: 60px;
+  list-style-position: inside;
+}
+
+.about-list li {
+  margin: 0 auto;
+  background-color: var(--secondary-color);
+  margin-bottom: 30px;
+  width: 570px;
+  border-color: transparent;
+  transition: all 0.3s ease-in-out;
+}
+
+.about-list li::marker {
+  content: counter(list-item, decimal-leading-zero);
+  color: var(--secondary-color);
+  font-size: 80px;
+  line-height: 96px;
+  text-shadow: 0px 0px 1px var(--btn-primary-color);
+}
+
+/* I couldn't add another angle border effect */
+.about-list li:hover {
+  border-width: 3px;
+  border-style: solid;
+  border-image: linear-gradient(
+      to bottom right,
+      rgba(33, 231, 134, 1) 0%,
+      rgba(33, 231, 134, 0) 20%
+    )
+    1;
+}
+
+.about-list article {
+  padding: 82px 30px 30px;
+  text-align: start;
+}
+
+.about-list h3 {
+  font-size: 24px;
+  line-height: 30px;
+  margin-bottom: 26px;
+}
+
+.about-list p {
+  width: 510px;
+}
+
+/* Workflow section */
+
+.workflow {
+  padding-top: 50px;
+  padding-bottom: 50px;
+  background-color: var(--main-background-color);
+}
+
+.workflow * {
+  background-color: transparent;
+}
+
+.workflow h2,
+.workflow span {
+  margin-bottom: 40px;
+}
+
+.workflow-cards {
+  /* gap: 30px; */
+  list-style-position: inside;
+  text-align: center;
+}
+
+.workflow-card {
+  margin: 0 auto;
+  background-color: var(--secondary-color);
+  margin-bottom: 30px;
+  padding: 30px;
+  width: 330px;
+}
+
+/* Change position of the marker and image */
+
+.workflow-card::marker {
+  content: "STEP " counter(list-item);
+  font-size: 22px;
+  line-height: 30px;
+  color: var(--btn-primary-color);
+}
+
+.workflow-card img {
+  margin: 0 auto;
+  margin-bottom: 20px;
+  display: block;
+}
+
+.workflow-card p {
+  font-size: 24px;
+  line-height: 32px;
+  text-wrap: nowrap;
+}
+
+.workflow-card:hover {
+  border-width: 3px;
+  border-style: solid;
+  border-image: linear-gradient(
+      to bottom right,
+      rgba(33, 231, 134, 1) 0%,
+      rgba(33, 231, 134, 0) 20%
+    )
+    1;
+}
+
+/* Road Map section */
+
+.roadmap {
+  padding-top: 50px;
+  padding-bottom: 50px;
+  background-color: var(--main-background-color);
+}
+
+.roadmap * {
+  background-color: transparent;
+}
+
+.roadmap h2 {
+  margin-bottom: 60px;
+}
+
+.roadmap-card {
+  background-color: var(--article-bg-color);
+  padding: 48px 43px;
+  margin-bottom: 25px;
+}
+
+.roadmap-img-box {
+  width: 715px;
+}
+
+.roadmap-card-title {
+  font-size: 36px;
+  line-height: 44px;
+  margin-bottom: 16px;
+}
+
+.roadmap-list {
+  margin-bottom: 30px;
+}
+
+.roadmap-items {
+  margin-bottom: 20px;
+  /* gap: 12px; */
+  font-family: var(--text-font);
+  color: var(--text-color);
+}
+
+.arrow-greater-btn {
+  float: inline-end;
```

Comment:
float is not the most appropriate property for positioning here:

The float property was originally designed for text wrapping around images, not for complex layouts. In modern web design, it's better to use Flexbox or Grid for layout and positioning, as they offer more control and are more predictable.

Reply:
I tried to fixed it. Not sure if it's a correct way. 

#fixed